pkg/daemon/worker.go

```diff
@@ -30,10 +29,11 @@ func (d *Daemon) Worker() *combined.Worker {
 		})
 	}
 
-	var sta stack.Interface
+	var pol *policy.Policy
 	{
-		sta = stack.New(stack.Config{
+		pol = policy.New(policy.Config{
```

Comment:
All of the caching/decision making logic moved into the policy package. This package was formerly an operator function and is now just a generic implementation of reusable code used within various operator functions.